ecl/hqlcpp/hqlhtcpp.cpp

```diff
@@ -5761,6 +5761,11 @@ bool isLibraryScope(IHqlExpression * expr)
 
 bool HqlCppTranslator::prepareToGenerate(HqlQueryContext & query, WorkflowArray & actions, bool isEmbeddedLibrary)
 {
+    query.expr.setown(expandDelayedFunctionCalls(&queryErrorProcessor(), query.expr));
```

Comment:
Will the be any problems in expanding DelayedFunctionCalls so early? it seems that functions were expanded much later in the previous version.

Reply:
No, it is deliberate so that the expression trees used with and without --fastsyntax are as similar as possible - eventually aiming to allow --fastsyntax when generating code.  Previously the expansion was too late.